improver/wxcode/wxcode_decision_tree_global.py

```diff
@@ -53,13 +53,12 @@ def wxcode_decision_tree_global():
               conditions should be combined;
               e.g. rainfall > 0.5 AND snowfall > 0.5
         * diagnostics_fields: The diagnostics which are being used in the
-              query. If this is a list of lists, the two fields in a given
-              list are subtracted (1st - (2nd * gamma)) and then compared
-              with the probability threshold.
-        * diagnostic_gamma (NOT UNIVERSAL): This is the gamma factor that
-              is used when comparing two fields directly, rather than
-              comparing a single field to a probability threshold.
-              e.g. gamma * P(SnowfallRate) < P(RainfallRate).
+              query. If this is a list of lists, the fields are manipulated
+              according to the operators in the list and then compared
+              with the probability threshold. In some cases, a factor of 0.7 is
```

Comment:
I put it in here to explain its purpose. I could make the language more abstract if you like: "For example, a factor of 0.7 would allow an assymmetric..."

Reply:
I'm not sure it's worth worrying about. Happy to leave it as it is